src/azul/project/hca/metadata_generator.py

```diff
@@ -108,17 +114,21 @@ def _get_schema_name_from_object(obj: JSON):
     def add_bundle(self,
                    bundle_uuid: str,
                    bundle_version: str,
+                   manifest,
                    metadata_files: List[JSON]) -> None:
 
-        file_uuids = self._get_file_uuids_from_objects(metadata_files)
+        file_info = self._get_file_uuids_from_objects(manifest, metadata_files)
 
-        for file, content in file_uuids.items():
+        for file, content in file_info.items():
```

Comment:
If the key isn't needed then no need to fetch the key
```suggestion
        for content in file_info.values():
```